frontend/static/js/pages/profilePage.js

```diff
@@ -5,10 +5,11 @@ import profileImg from '../components/profileComponents/profileImg.js';
 import profileTitle from '../components/profileComponents/profileTitle.js';
```

Comment:
mm I'd prefer no changes in this file, only additions if need be!

Reply:
Reverted